src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -634,16 +633,13 @@ private class ACCT_IndividualAccounts_TEST {
 
         assertAccount = [SELECT Id, RecordType.Name, Name
                          FROM Account];
-
```

Comment:
@kdy618, looks like there is some inconsistency on empty lines between query and assertion code.